docs/intro.md

```diff
@@ -545,6 +576,241 @@ By default the partner chain node process will look for key stores in the base p
 or your particular Partner Chain's documentation for information on how
 to manage your node keys.
 
+### Partner Chain Commands
+The Partner Chain toolkit provides a range of commands to interact with the chain or carry out
+administrative tasks. Note that all commands also accept a set of general options which can be
+listed by passing `--help` to the respectvie command.
+
+#### registration-status
+Returns registration status for a given mainchain public key and epoch number. If registration
+has been included in Cardano block in epoch N, then it should be returned by this command if
+epoch greater than N+1 is provided. If this command won't show your registration after a few
+minutes after it has been included in a cardano block, you can start debugging for unsuccessful
+registration.
+
+```shell
+$ pc-node registration-status
+    --stake-pool-pub-key <STAKE_POOL_PUB_KEY>
+    --mc-epoch-number <MC_EPOCH_NUMBER>
+```
+#### ariadne-parameters
+Returns ariadne parameters effective at given mainchain epoch number. Parameters are effective
+two epochs after the block their change is included in.
+
+```shell
+$ pc-node ariadne-parameter --mc-epoch-number <MC_EPOCH_NUMBER>
+```
+
+#### registration-signatures
+Generates registration signatures for partner chains committee candidates.
+
+```shell
+$ pc-node registration-signatures --genesis-utxo <GENESIS_UTXO>
+    --mainchain-signing-key <MAINCHAIN_SIGNING_KEY>
+    --sidechain-signing-key <SIDECHAIN_SIGNING_KEY>
+    --registration-utxo <REGISTRATION_UTXO>
+```
+#### sign-address-association
+Signs address association
+
+```shell
+$ pc-node sign-address-association
+    --genesis-utxo <GENESIS_UTXO>
+    --partnerchain-address <PARTNERCHAIN_ADDRESS>
+    --signing-key <SIGNING_KEY>
+```
+#### sign-block-producer-metadata
+Signs block producer metadata for submitting to the runtime.
+
+```shell
+$ pc-node sign-block-producer-metadata
+    --genesis-utxo <GENESIS_UTXO>
+    --metadata-file <METADATA_FILE>
+    --cross-chain-signing-key <CROSS_CHAIN_SIGNING_KEY>
+```
+#### smart-contracts
+The smart contracts command provides multiple sub-commands for interacting with Partner Chain smart
+contracts on Cardano. Note that all of the smart-contract commands require the `ogmios` service to
+be running locally.
+##### get-scripts
+Prints validator addresses and policy IDs of Partner Chain smart contracts
+
+```shell
+$ pc-node smart-contracts get-scripts --genesis-utxo <GENESIS_UTXO>
+```
+
+##### upsert-d-parameter
+Upsert D-parameter
```

Comment:
I'm not sure if it isn't enough to enumerate governance related smart contracts without additional documentation here. IMO these comments don't add value.

Reply:
100% right and agreed. That's the very reason why the PR is still in DRAFT. Whenever the documentation only repeats the name of the command that really doesn't help a lot